spiffworkflow-backend/src/spiffworkflow_backend/services/user_service.py

```diff
@@ -82,6 +83,14 @@ def current_user() -> Any:
             raise ApiError("logged_out", "You are no longer logged in.", status_code=401)
         return g.user
 
+    @classmethod
+    def is_logged_in_as_guest_user(cls) -> bool:
```

Comment:
i think we can kill this unless it's being used by something meta-programmed.

Reply:
Looks like not. I think was using it but seems no longer necessary.